packages/manager/apps/dedicated/client/app/billing/payment/method/add/translations/Messages_fr_FR.json

```diff
@@ -27,5 +27,7 @@
   "billing_payment_method_add_vantiv_recoverable_credit_card_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement. Veuillez vérifier le numéro de votre carte bancaire puis le ressaisir.",
   "billing_payment_method_add_vantiv_recoverable_card_validation_number_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement. Veuillez vérifier le numéro de validation de votre carte bancaire puis le ressaisir.",
   "billing_payment_method_add_success": "Votre nouveau moyen de paiement a bien été ajouté.",
-  "billing_payment_method_add_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement : {{ errorMessage }}"
+  "billing_payment_method_add_error": "Une erreur est survenue lors de l'ajout de votre moyen de paiement : {{ errorMessage }}",
+  "billing_payment_method_add_alert_add_method": "Si vous rencontrez un problème pour ajouter un nouveau moyen de paiement, veuillez ajouter une exception à votre bloqueur de publicités pour {{ hostname }}, ou veuillez vous connecter via le mode incognito de votre navigateur, puis réessayez. Nous n'afficherons jamais de publicités sur notre site web. Sachez que votre bloqueur de publicités peut interférer avec notre système de paiement.",
```

Comment:
Not mandatory but i would sightly change the sentence to be more specific to the manager :
"nous n'afficherons jamais de publicités ~~sur notre site web~~ dans l'interface client" ?

Reply:
Why not, but it is translated from given EN content, which has "our website" sentence

Reply:
@JeremyDec, maybe english content should be "our control panel"